docs/guides/catalog.rst

```diff
@@ -402,3 +402,130 @@ Querying by Space and Time
               layer_zoom=7,
               query_geom=multi_poly,
               time_intervals=[min_key.instant, max_key.instant])
+
+AttributeStore
+--------------
+
+When writing a layer GeoTrellis uses an ``AttributeStore`` to write layer metadata required to read and query the layer later.
+This class can be used outside of catalog ``write`` and ``query`` functions to inspect available layers and store additional, user defined, attributes.
+
+Creating AttributeStore
+~~~~~~~~~~~~~~~~~~~~~~~
+
+``AttributeStore`` can be created from the same ``URI`` that is given to ``write`` and ``query`` functions.
```

Comment:
It might be good to have links to the `AttributeStore`'s [class docs](https://geopyspark.readthedocs.io/en/latest/docs/geopyspark.geotrellis.html#geopyspark.geotrellis.catalog.AttributeStore). We do this throughout the rest of the docs.

The following is how you'd create a link to that in the doc:
```
:class:`~geopyspark.geotrellis.catalog.AttributeStore`
```